Storage/ThirdParty/SwiftData.swift

```diff
@@ -267,6 +267,8 @@ private class SQLiteDBStatement {
             } else if obj is Date {
                 let timestamp = (obj as! Date).timeIntervalSince1970
                 status = sqlite3_bind_double(pointer, Int32(index+1), timestamp)
+            } else if obj is UInt64 {
```

Comment:
It strikes me as more Swifty to parse a `UInt64` directly than to use an `NSNumber` to add to an `Any` in order to convert a `UInt64` into a `Double`.

I went through this several times. If the obj is a UInt64, a Timestamp, then the value doesn't get bound. Then, when the statement is executed, an error is thrown becuase `last_modified` is a nil object. To be honest, I have no idea how this ever worked, if a previous version also passed in a UInt64.